stdlib/3/concurrent/futures/_base.pyi

```diff
@@ -46,14 +46,17 @@ class Future(Generic[_T]):
         def set_exception_info(self, exception: Any, traceback: Optional[TracebackType]) -> None: ...
 
 class Executor:
-    def submit(self, fn: Callable[..., _T], *args: Any, **kwargs: Any) -> Future[_T]: ...
+    def submit(self, __fn: Callable[..., _T], *args: Any, **kwargs: Any) -> Future[_T]: ...
```

Comment:
I don't think that changing it to a positional-only argument is correct, at least not without a version check. In 3.8.5, there is an explicit check for a `fn` argument. While a warning is generated, it is accepted. Of course, it's a bit weird, since the checking code is within an abstract method, so it will likely never get executed.

Reply:
Good catch, weird the documentation doesn't mentioned the break in backwards compatibility.